index.bs

```diff
@@ -719,63 +719,80 @@ Putting these rules in words:
+    |config|[{{SanitizerConfig/replaceWithChildrenElements}}] is not [=list/empty=], then return
+    false.
+1. If |config|[{{SanitizerConfig/removeElements}}] [=map/exists=] and
+    |config|[{{SanitizerConfig/replaceWithChildrenElements}}] [=map/exists=]
     the [=SanitizerConfig/intersection=] of |config|[{{SanitizerConfig/removeElements}}] and
-    |config|[{{SanitizerConfig/replaceWithChildrenElements}}] is [=list/empty=].
+    |config|[{{SanitizerConfig/replaceWithChildrenElements}}] is not [=list/empty=], then return
+    false.
 1. If |config|[{{SanitizerConfig/attributes}}] [=map/exists=]:
     1. If |config|[{{SanitizerConfig/elements}}] [=map/exists=]:
         1. [=list/iterate|For each=] |element| of |config|[{{SanitizerConfig/elements}}]:
-            1. Neither |element|[{{SanitizerElementNamespaceWithAttributes/attributes}}] nor
-               |element|[{{SanitizerElementNamespaceWithAttributes/removeAttributes}}], if they exist, [=SanitizerConfig/has duplicates=].
-            1. The [=set/intersection=] of |config|[{{SanitizerConfig/attributes}}] and
+            1. If |element|[{{SanitizerElementNamespaceWithAttributes/attributes}}] [=map/exists=]
+                and |element|[{{SanitizerElementNamespaceWithAttributes/attributes}}]
+                [=SanitizerConfig/has duplicates=], then return false.
+            1. If |element|[{{SanitizerElementNamespaceWithAttributes/removeAttributes}}]
+                [=map/exists=] and
+                |element|[{{SanitizerElementNamespaceWithAttributes/removeAttributes}}]
+                [=SanitizerConfig/has duplicates=], then return false.
+            1. If the [=set/intersection=] of |config|[{{SanitizerConfig/attributes}}] and
                 |element|[{{SanitizerElementNamespaceWithAttributes/attributes}}] [=with default=]
-                &laquo; &raquo; is [=list/empty=].
-            1. |element|[{{SanitizerElementNamespaceWithAttributes/removeAttributes}}]
-                [=with default=] &laquo; &raquo; is a
-                [=set/subset=] of |config|[{{SanitizerConfig/attributes}}].
+                &laquo; &raquo; is not [=list/empty=], then return false.
+            1. If |element|[{{SanitizerElementNamespaceWithAttributes/removeAttributes}}]
+                [=with default=] &laquo; &raquo; is not a
+                [=set/subset=] of |config|[{{SanitizerConfig/attributes}}], then return false.
             1. If {{SanitizerConfig/dataAttributes}} [=map/exists=] and
-                {{SanitizerConfig/dataAttributes}} is true:
-                1. |element|[{{SanitizerElementNamespaceWithAttributes/attributes}}] does not
-                    contain a [=custom data attribute=].
+                {{SanitizerConfig/dataAttributes}} is true and
+                |element|[{{SanitizerElementNamespaceWithAttributes/attributes}}]
+                contains a [=custom data attribute=], then return false.
```

Comment:
I can't make a suggestion here, but now the condition has "and" twice. First "and" should be replaced with a comma and the second needs a comma before it.

Reply:
Done.